web.test.tests/web.test.tests/Pages/DepositPage.cs

```diff
@@ -117,5 +127,38 @@ public void SelectFinYear(int year)
                     throw new Exception($"Not valid financial year value: {year}");
             }
         }
+        public void Calculate()
+        {
+            CalculateBtn.Submit();
+            Thread.Sleep(500);
+        }
+
+        public Dictionary<String, String> CaptureData()
+        {
+            String year = "";
+            if (FinYear360.GetAttribute("checked") == "true") year = "360";
+            if (FinYear365.GetAttribute("checked") == "true") year = "365";
```

Comment:
There is 'ternary operator', you may use it in such situations:

```suggestion
            String year = FinYear360.GetAttribute("checked") == "true" ? "365" : "360";
```

Reply:
Corrected